neuraxle/pipeline.py

```diff
@@ -382,8 +382,10 @@ def _load_checkpoint(self, data_container: DataContainer) -> Tuple[NamedTupleLis
         new_starting_step_index, starting_step_data_container = self._find_starting_step_index(data_container)
 
         if self.parent_step is None:
-            pipeline_cache = self.pipeline_repository.load(self, starting_step_data_container)
-            loaded_pipeline = self._try_loading_pipeline_checkpoint(pipeline_cache, new_starting_step_index)
+            loaded_pipeline = self._try_loading_pipeline_checkpoint(
```

Comment:
boolean variable must start with `is_` or `has_`, etc., otherwise here I think this is not a bool, especially that a try method would return an object in my mind, not a bool.

Reply:
it has a side effect and it returns true if it succeeds doing it. hence why I named it try loading pipeline checkpoint... not sure how I would rename it here

Reply:
renamed it to _load_saved_pipeline it returns true if the loading suceeded